addons/mail/static/src/components/chat_window_header/chat_window_header.scss

```diff
@@ -47,29 +30,15 @@
     }
 }
 
-.o_ChatWindowHeader_name {
-    max-height: map-get($sizes, 100);
-}
-
-.o_ChatWindowHeader_rightArea {
-    display: flex;
-    height: map-get($sizes, 100);
-    align-items: center;
-}
-
 // ------------------------------------------------------------------
 // Style
 // ------------------------------------------------------------------
 
 .o_ChatWindowHeader {
     background-color: $o-brand-odoo;
-    color: $white;
 
     &:not(.o-mobile) {
         cursor: pointer;
-    }
-
-    &:not(.o-mobile) {
 
         &:hover .o_ChatWindowHeader_command {
             opacity: 0.7;
```

Comment:
You can use combinations of the new opacity classes dihttps://github.com/odoo/odoo/blob/master/addons/web/static/src/legacy/scss/ui.scss#L49

Reply:
Done, but not exactly the same result. 
Currently, the icons on the right go from 50% to 75% opacity on hovering anywhere in the chatHeader area.
With this change, the icons on the right go from 50% to 75% opacity on hovering the rightArea of the chatHeader: on hovering an icon, that icon has opacity 100%, the others have opacity 75%, but nothing happens if I hover anywhere else in the header.

Reply:
Gotha. It's feasible for me, the behaviour is maintained even if it's slightly different... we'll check with seb later on, just in case I'm missing something 👍